src/App.js

```diff
@@ -33,10 +66,19 @@ class Image extends React.Component {
         window.removeEventListener("scroll", this.handleScroll);
     }
 
+    deleteCat = () => {
+        const countToDelete = this.state.countToDelete
```

Comment:
_minor_: you may check if `countToDelete` equals to `0` and do nothing in this case.